ui/raidboss/data/05-shb/alliance/the_tower_at_paradigms_breach.txt

```diff
@@ -0,0 +1,231 @@
+2199.3 "Tandem Assault: Breakthrough" sync /:Gretel:61BA:/
+2201.4 "--untargetable--"
+2201.5 "--sync--" sync /:Gretel:5C7E:/
+2212.8 "Breakthrough" sync /:Hansel:5C5E:/
+2213.1 "Uneven Footing" sync /:Hansel & Gretel:5C5F:/
+2214.9 "--targetable--"
+
+2227.0 "Upgraded Shield" sync /:Gretel:5C6[89]:/
+2227.0 "Upgraded Lance" sync /:Hansel:5C6[AB]:/
+2240.2 "Wail" sync /:Hansel:5C77:/
+2240.3 "Crippling Blow (G)" sync /:Gretel:5C78:/
+2250.5 "Wail" sync /:Gretel:5C76:/
+2250.6 "Crippling Blow (H)" sync /:Hansel:5C79:/
+
+2262.9 "Tandem Assault: Passing Lance" sync /:Gretel:61BC:/
+2266.1 "--untargetable--"
+2266.2 "--sync--" sync /:Gretel:5C7E:/
+2267.9 "Seed Of Magic Alpha" sync /:Gretel:5C61:/
+2267.9 "Riot Of Magic" sync /:Hansel:5C63:/
+2276.4 "Passing Lance" sync /:Gretel:5C64:/
+2281.2 "--targetable--"
+
+2291.3 "Upgraded Shield" sync /:Gretel:5C6[89]:/
+2291.3 "Upgraded Lance" sync /:Hansel:5C6A:/
+2307.6 "Wandering Trail" sync /:Gretel:5C5A:/
+2315.8 "--sync--" sync /:Gretel:5C7E:/
+2322.5 "Hungry Lance" sync /:Gretel:5C71:/
+
+2334.7 "Upgraded Shield" sync /:Gretel:5C6[89]:/
+2334.7 "Upgraded Lance" sync /:Hansel:5C6[AB]:/
+2351.0 "--sync--" sync /:Gretel:5C7C:/ # Seed Of Magic Beta cast
+2353.2 "--sync--" sync /:Gretel:5C7E:/
+2359.9 "Wandering Trail" sync /:Gretel:5C5A:/
+2361.8 "Seed Of Magic Beta" sync /:Gretel:5C75:/ # damage
+
+2367.1 "Tandem Assault: Breakthrough" sync /:Gretel:61BA:/
+2369.2 "--untargetable--"
+2369.3 "--sync--" sync /:Gretel:5C7E:/
+2380.6 "Breakthrough" sync /:Hansel:5C5E:/
+2380.9 "Uneven Footing" sync /:Hansel & Gretel:5C5F:/
+2382.7 "--targetable--"
+
+2394.8 "Upgraded Shield" sync /:Gretel:5C6[89]:/
+2394.8 "Upgraded Lance" sync /:Hansel:5C6[AB]:/
+2408.1 "Wail" sync /:Hansel:5C77:/
+2408.2 "Crippling Blow (G)" sync /:Gretel:5C78:/
+2418.4 "Wail" sync /:Gretel:5C76:/
+2418.5 "Crippling Blow (H)" sync /:Hansel:5C79:/
+
+2430.8 "Tandem Assault: Bloody Sweep" sync /:Gretel:61B8:/
+2433.0 "--sync--" sync /:Gretel:5C7E:/
+2435.3 "Tandem" sync /:Gretel:5C58:/
+2445.5 "Transference" sync /:Gretel:5CF1:/
+2450.4 "Bloody Sweep" sync /:Gretel:5C56:/
+2464.6 "Upgraded Shield" sync /:Gretel:5C6[89]:/
+2464.6 "Upgraded Lance" sync /:Hansel:5C6[AB]:/
+
+# loop
+2476.9 "Tandem Assault: Passing Lance" sync /:Gretel:61BC:/ window 100,100 jump 2262.9
+2480.1 "--untargetable--"
+2480.2 "--sync--" #sync /:Gretel:5C7E:/
+2481.9 "Seed Of Magic Alpha" #sync /:Gretel:5C61:/
+2481.9 "Riot Of Magic" #sync /:Hansel:5C63:/
+2490.4 "Passing Lance" #sync /:Gretel:5C64:/
+2495.2 "--targetable--"
+
+2505.3 "Upgraded Shield" #sync /:Gretel:5C6[89]:/
+2505.3 "Upgraded Lance" #sync /:Hansel:5C6A:/
+2521.6 "Wandering Trail" #sync /:Gretel:5C5A:/
+2529.8 "--sync--" #sync /:Gretel:5C7E:/
+2536.5 "Hungry Lance" #sync /:Gretel:5C71:/
+
+
+# Enrage (when one dies)
+2700.0 "--sync--" sync /:5C73:Gretel/ window 700,0
+2700.0 "--sync--" sync /:5C74:Hansel/ window 700,0
+2708.0 "Lamentation" sync /:Gretel:5C73:/
```

Comment:
At least when I ran this, Lamentation is just a repeated raidwide AoE, much like O12. It's possible there's an actual enrage after X number of Lamentations, but I got hit I think four times and survived.

Reply:
Do you have a timeline snippet for that portion?

Reply:
```
438.5 "Lamentation" sync /:Hansel:5C74:/
444.3 "Seed Of Magic Beta" sync /:Hansel:5C75:/
448.6 "Lamentation" sync /:Hansel:5C74:/
454.4 "Seed Of Magic Beta" sync /:Hansel:5C75:/
458.7 "Lamentation" sync /:Hansel:5C74:/
```

It looks like I misremembered, Hansel died before beginning the fourth cast. However, I think this is *probably good enough to loop with unless/until someone gives us an enrage log. (We might be able to find one on FFLogs soon?)

Reply:
> I'll have to dig into it more tomorrow, but we might need more data surrounding Colossal Impact:

Hmm, thanks for this.  I think I'll have to gather more logs before I understand what's going on here.  It may be a case of the adds being desynced or something else going on here.